projects/lendai-us/design-system/.storybook/styles.scss

```diff
@@ -1,6 +1,41 @@
 @use '@angular/material' as mat;
 @use 'theme' as t;
+@use 'color' as c;
+@use 'sass:map';
+@import 'theme.scss';
```

Comment:
prefer using `@use` over `@import` since `@import` is deprecated now 

https://stackoverflow.com/questions/62757419/whats-the-difference-between-import-and-use-scss-rules